### PR TITLE
DOC-6343 Add documentation for running multiple single node containers

### PR DESCRIPTION
**Pull request by RayOffiah**
https://issues.couchbase.com/browse/DOC-6343

### CODE REVIEW

**Comment by RayOffiah** [comment]
Preview at:

https://rayoffiah.github.io/DOC-6343/server/current/install/getting-started-docker.html#section_deploy_multiple_clusters

**Review by tonyjhillman — APPROVED** [review]

File: modules/install/pages/getting-started-docker.adoc [hunk 1]
```diff
@@ -323,6 +325,130 @@ image::docker-single-machine-db123.png["The 'Servers' tab showing three nodes in
...
+$ docker run -d --name db1 -p 8091-8096:8091-8096 -p 11210-11211:11210-11211 couchbase
+----
++
+[source,console]
+----
+$ docker run -d --name db2 -p 9091-9096:8091-8096 -p 12210-12211:11210-11211 couchbase
+----
++
+After running the above commands,
+three instances (`db1` and `db2`) of the latest https://hub.docker.com/_/couchbase/[official Couchbase Server container image^] are downloaded
```
Comment (malarky) [inline modules/install/pages/getting-started-docker.adoc]: Should be "two instances".

File: modules/install/pages/getting-started-docker.adoc [hunk 2]
```diff
@@ -323,6 +325,130 @@ image::docker-single-machine-db123.png["The 'Servers' tab showing three nodes in
...
+and installed onto the host computer.
```
Comment (malarky) [inline modules/install/pages/getting-started-docker.adoc]: I wouldn't say "installed" when talking about a container.

File: modules/install/pages/getting-started-docker.adoc [hunk 3]
```diff
@@ -323,6 +325,130 @@ image::docker-single-machine-db123.png["The 'Servers' tab showing three nodes in
...
++
+image::welcome-join-existing-cluster.png["The Couchbase Server setup screen.",400]
+. Click btn:[Join Existing Cluster].
+
++
+image::join-cluster-details.png[Joining an existing cluster, 450]
++
+. Fill in the IP address associated with container `db1`, along with the administrator password.
++
+. Press btn:[Join Cluster]
```
Comment (malarky) [inline modules/install/pages/getting-started-docker.adoc]: I though this section was about running multiple single-node clusters, so shouldn't be any discussion around adding nodes and rebalance.

File: modules/install/pages/getting-started-docker.adoc [hunk 4]
```diff
@@ -323,6 +325,130 @@ image::docker-single-machine-db123.png["The 'Servers' tab showing three nodes in
...
+Make sure to run each of the following commands:
++
+[source,console]
+----
+$ docker run -d --name db1 -p 8091-8096:8091-8096 -p 11210-11211:11210-11211 couchbase
+----
++
+[source,console]
+----
+$ docker run -d --name db2 -p 9091-9096:8091-8096 -p 12210-12211:11210-11211 couchbase
```
Comment (malarky) [inline modules/install/pages/getting-started-docker.adoc]: You can't re-use `12210-12211:11210-11211`, need to find a new set as you did with `8091`.

Comment (malarky) [inline modules/install/pages/getting-started-docker.adoc]: Also, I don't think a client will be able to connect unless you set up Alternate Addresses.

**Review by malarky — CHANGES_REQUESTED** [review]
Some specific comments, but also wondering if we just need to provide some general guidance on running multiple instances, regardless of how they are clustered.

I *think* (but you should test and verify), if you only want to get to the UI then you can get away with just using the `-p` for 8091 and incrementing the host port.  But, if you actually want an application to connect you'll have to set up alternate addresses for every container after the first one.

If you go over the basics (use of `-p` and alternate addresses), then users can set up as many containers as they want and then cluster them how they want.

File: modules/install/pages/getting-started-docker.adoc (hunk 1 above)
Comment (RayOffiah) [reply modules/install/pages/getting-started-docker.adoc]: Fixed throughout document

**Pull request merged**